offchain/danaswap-browser/src/DanaSwap/Browser/Home.purs

```diff
@@ -19,18 +24,239 @@ main = HA.runHalogenAff do
     Nothing -> throwError $ error "Cannot find #home"
```

Comment:
Nit and skip if you don't like, but this error message may be more clear:

```
`#home` element to found. DanaSwap component unable to mount.
```

Open to rewording, but the current message is a bit too non-specific.